boofuzz/sessions.py

```diff
@@ -685,6 +688,13 @@ def _main_fuzz_loop(self, fuzz_case_iterator):
             self._fuzz_data_logger.close_test()
             if self._reuse_target_connection:
                 self.targets[0].close()
+
+            if self._keep_web_open and self.web_port is not None:
+                print("\nFuzzing session completed. Keeping webinterface up on localhost:{}".format(self.web_port),
+                      "\nPress ENTER to close webinterface")
+                raw_input()
+            else:
+                print("\nFuzzing session completed. Webinterface will be available as long as this script is running.")
```

Comment:
I'd lean toward omitting the else statement -- in the normal case the script will exit right after it's printed, right?

Reply:
Yes, as long as the script ends after `sess.fuzz()`. The session itself won't close the web-thread as it is a daemon and will run until the main thread exits. My idea was to make the user aware of the web-thread still running as long as the script is too. I see that there are practically no cases where that could happen though, so I'm just going to remove it.

For #259 it might help to manually close the web-thread when fuzz() is completed.